keps/sig-contributor-experience/0000-community-forum/kep.yaml

```diff
@@ -0,0 +1,16 @@
+title: A community forum for Kubernetes
```

Comment:
Can we open some placeholder issues for these to avoid 0000?

This would make strenghtening validating more powerful.

Reply:
We discussed this a bit in slack and at the last enhancements meeting. The conclusion was to use 0000 (as you can't have a github issue with 0) for items that were worked and implemented before the k/community -> k/enhancements migrations.

Now that I think about it though, the contrib site one could be updated with an issue. It didn't go live till this year after significant rescoping/re-design.

https://kubernetes.slack.com/archives/C1L57L91V/p1608571448268500

Reply:
I guess I'm tentatively fine if there won't be too many of them and we can add some exceptions to them...

[Though only for the ones that were really implemented - there seem to be quite a bit of such ones in provisional state...]

Reply:
I've updated the contributor site kep with an issue. After this is merged I'll take care of updating it with the implementation history and details

Reply:
Thanks!